private/buf/cmd/buf/buf_test.go

```diff
@@ -612,6 +612,207 @@ func TestCheckLsLintRulesFromConfig(t *testing.T) {
 		"--config",
 		filepath.Join("testdata", "small_list_rules", "buf.yaml"),
 	)
+	// defaults only, built-ins and plugins.
+	testRunStdout(
+		t,
+		nil,
+		0,
+		`
+ID                                CATEGORIES                DEFAULT  PURPOSE
```

Comment:
Just a general note: these tests are going to be an absolute bear to maintain. The existing `ls-lint-rules` tests across three config versions is OK, and has saved me a few times, but having ~10 of these is going to be tough. I don't know if there's a time-efficient way to refactor, perhaps with `contains`-type tests, to test just what you want. Also `--format-json` flag might be helpful. But perhaps not worth it.

Reply:
This is addressed in [84df89c](https://github.com/bufbuild/buf/pull/3277/commits/84df89c5ffa7e2ab3906dcf406eddb4c46b986f9), making the new tests json.

Note: there are 9 tests from the past testing the table output.